hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/app.tsx

```diff
@@ -70,7 +71,7 @@ class App extends React.Component<Record<string, object>, IAppState> {
           <Layout className={layoutClass}>
             <Header>
```

Comment:
If we move to the old insights page and click on the UI change button we get en empty page 

<img width="953" alt="image" src="https://github.com/user-attachments/assets/89b4643a-7f10-4cc1-a79d-1ca285e6b1b6">

Reply:
This is expected behaviour since in the new UI we are changing the URL to OmInsights.
Reverting this back for now and will update when we completely shift to V2 UI.
Addressed in [8add6c2](https://github.com/apache/ozone/pull/7327/commits/8add6c2fccf9b0e999fac47ef0a65fa19e996b2c)